ultra/competition/starting_kit/README.md

```diff
@@ -0,0 +1,351 @@
+
+The `stop` and `bubbles` are higher-level features that allow you to specify stopped vehicles and utilize zoo agents in your scenario, respectively. We imagine that these will not necessarily be needed for the competition, however, ways to implement them can be found in the main ULTRA code in the [SMARTS repository](https://github.com/huawei-noah/SMARTS).
+
+A level can have multiple intersection types. You can find an example of a task's `config.yaml` under `scenarios/example_scenarios/`.
+
+### Building scenarios
+First, build the maps used in the scenarios. This step has to be done only once at the beginning, and whenever the maps change.
+
+```bash
+$ scl scenario build-all scenarios/pool/
+```
+
+Assume we have a task called `my_task`, for which a `config.yaml` is available with a level name "`my_level`". Create a new directory structure as follows:
+```
+scenarios
+└── my_task
+    └── config.yaml
+```
+
+Next, run the `scenarios/build_scenarios.py` script which will read our `config.yaml` and create the `my_level` scenarios:
+
+```bash
+$ python scenarios/build_scenarios.py --task my_task --level my_level --save-dir scenarios/my_task/ --root-dir scenarios/ --pool-dir scenarios/pool/
+```
+
+Now, you should see scenarios for your `my_level` under `scenarios/my_task/`.
+
+The arguments for `build_scenarios.py` are as follows:
+- `--task`: Name of the task to build. It should match a directory name under the specified `--root-dir` argument.
+- `--level`: Level to build from the task's `config.yaml`.
+- `--save-dir`: Directory in which to save the created scenarios.
+- `--root-dir`: Directory containing the task's folder.
+- `--pool-dir`: Location of the maps used to build the scenarios.
+
+## Submitting an agent for evaluation
+1. Ensure you have an `agent.py` file and an `agent_spec` variable in that file
+
+   Your agent submission should contain at least one file called `agent.py`. This file should contain a variable called `agent_spec` that references a `smarts.core.agent.AgentSpec` instance. The `agent_spec` defines how your agent will be built.
+
+1. If the agent is built upon a baseline, ensure that the agent has access to your desired checkpoint.
```

Comment:
```suggestion
2. If the agent is built upon a baseline, ensure that the agent has access to your desired checkpoint.
```

Reply:
Sequence of `1.` in a `.md` file will automatically appear as proper numbered sequence when displayed by a reader which supports markdown files. This makes numbering easier for documentation maintainers.

I think I will change it to `1,2,3,4,5` sequence, just in case CodaLab does not interpret markdown files properly.